src/pages/publication/PublicProfile.tsx

```diff
@@ -31,68 +28,60 @@ const StyledUserInfo = styled.div`
-      }
-      setIsLoading(false);
-    };
-    loadData();
-  }, [dispatch, t]);
-
-  const publishedPublications = publications.filter(
-    (publication) => publication.status === PublicationStatus.PUBLISHED
-  );
+  const { arpId } = useParams();
+  const [authority, isLoadingUser] = useFetchAuthority(arpId);
 
   return (
     <>
-      <StyledUserInfo>
-        <Card>
-          <Heading>{user.name}</Heading>
-          {user.authority?.orgunitids && user.authority.orgunitids.length > 0 && (
-            <LabelTextLine label={t('heading.organizations')}>
-              {user.authority.orgunitids.map((unitId) => (
-                <AffiliationHierarchy key={unitId} unitUri={unitId} commaSeparated />
-              ))}
-            </LabelTextLine>
-          )}
-          {user.authority?.orcids.map((orcid: string) => {
-            const orcidLink = `${ORCID_BASE_URL}/${orcid}`;
-            return (
-              <LabelTextLine
-                key={orcid}
-                dataTestId={'orcid-info'}
-                label={t('orcid.orcid')}
-                linkText={orcidLink}
-                externalLink={orcidLink}
-              />
-            );
-          })}
-          <LabelTextLine dataTestId="profile-email" label={t('common:email')}>
-            <NormalText>{user.email}</NormalText>
-          </LabelTextLine>
-        </Card>
-      </StyledUserInfo>
-      <StyledWrapper>
-        {isLoading ? (
-          <CircularProgress color="inherit" size={20} />
-        ) : (
-          <PublishedPublicationList publications={publishedPublications} />
-        )}
-      </StyledWrapper>
+      {isLoadingUser ? (
+        <CircularProgress />
+      ) : (
+        authority && (
+          <>
+            <StyledUserInfo>
+              <Card>
+                <Heading>{authority.name}</Heading>
+                {authority.orgunitids.length > 0 && (
+                  <LabelTextLine label={t('heading.organizations')}>
+                    {authority.orgunitids.map((unitId) => (
+                      <AffiliationHierarchy key={unitId} unitUri={unitId} commaSeparated />
+                    ))}
+                  </LabelTextLine>
+                )}
+                {authority?.orcids.map((orcid: string) => {
+                  const orcidLink = `${ORCID_BASE_URL}/${orcid}`;
+                  return (
+                    <LabelTextLine
+                      key={orcid}
+                      dataTestId={'orcid-info'}
+                      label={t('orcid.orcid')}
+                      linkText={orcidLink}
+                      externalLink={orcidLink}
+                    />
+                  );
+                })}
+              </Card>
+            </StyledUserInfo>
+            <StyledWrapper>
+              <PublicProfilePublications arpId={arpId} />
+            </StyledWrapper>
+          </>
+        )
+      )}
     </>
   );
 };
 
+interface PublicProfilePublicationsProps {
+  arpId: string;
+}
```

Comment:
nødvendig å deklarere et eget interface for en subklasse med kun en prop? kanskje bare definere den propen direkte?

Reply:
etter hva som ble sagt på planning tenker jeg vi bare kan fjerne `PublicProfilePublications` for nå siden det bare er en placeholder for noe som ikke kommer på en stund uansett

Reply:
Ja, kan godt gjøre det 👍